react/components/EditorContainer/Sidebar/EditorHeader/index.tsx

```diff
@@ -23,7 +23,7 @@ const EditorHeader: React.SFC<Props> = ({
   title,
 }) => (
   <div
-    className={`w-100 pl5 pt5 pb3 bb bw1 b--light-silver ${
+    className={`w-100 pl5 pv4  bb bw1 b--light-silver pr5 ${
       !onSave ? 'pr5' : ''
```

Comment:
Since we're always using `pr5`, there's no need to have this check anymore. Also, `pl5` and `pr5` could just be `ph5`. Finally, there's an extra whitespace after `pv4`.

Applying all of the aforementioned changes, the `className` prop would become:
```js
className="w-100 ph5 pv4 bb bw1 b--light-silver"
```